src/combobox/combobox.component.ts

```diff
@@ -377,7 +377,6 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 				}
 				this.selected.emit(event);
 			});
-			this.view.items = this.items;
```

Comment:
How does removing this allow items to be updated via `setValue`/`writeValue`?

Reply:
It already was updating the values but this line has been overwriting the values that are set through set/write value

Reply:
Interesting 🤔